youtube_dl/extractor/youtube.py

```diff
@@ -1502,8 +1571,35 @@ def add_dash_mpd(video_info):
             if dash_mpd and dash_mpd[0] not in dash_mpds:
                 dash_mpds.append(dash_mpd[0])
 
+        def get_storyboards(video_info):
+            storyboards = []
+            spec = video_info.get('storyboard_spec', [])
+
+            for s in spec:
+                s_parts = s.split('|')
+                base_url = s_parts[0]
+                i = 0
+                for params in s_parts[1:]:
+                    w, h, frames, cols, rows, whoknows, filename, sigh = \
+                        map(lambda x: int(x) if x.isdigit() else x, params.split('#'))
```

Comment:
Breaks extraction completely if `params` does not match number of values to unpack.